gripper_bot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  arm_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_joint
    pid: {p: 0.2, i: 0.0, d: 0.5}
  gripper_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: gripper_joint
    pid: {p: 0.5, i: 0.0, d: 0.0}
